Tools/GrownBiomeGenVisualiser/Globals.h

```diff
@@ -191,7 +191,23 @@ typedef unsigned char Byte;
 
 
 // Common headers (without macros):
+
+#ifdef __clang__
+	#pragma clang diagnostic push
+	#pragma clang diagnostic ignored "-Wunknown-pragmas"
+	#pragma clang diagnostic ignored "-Wunknown-warning-option"
+	#pragma clang diagnostic ignored "-Wsigned-enum-bitfield"
+	#pragma clang diagnostic ignored "-Wundefined-func-template"
+	#pragma clang diagnostic ignored "-Wc++2a-compat"
+#endif
+
 #include "fmt/format.h"
+#include "fmt/printf.h"
+
+#ifdef __clang__
+	#pragma clang diagnostic pop
+#endif
+
```

Comment:
Wouldn't it be easier to put all this into one separate include file? `#include "fmt.h"`